### PR TITLE
Adding getLogos method to the client and type to the AlApplication interface

### PR DESCRIPTION
**Pull request by jleon-alertlogic**

### CODE REVIEW

File: src/al-applications-client.ts [hunk 1]
```diff
@@ -37,6 +37,28 @@ export class AlApplicationsClientInstance {
...
+     *
+     *  @param imageName this is the Id of the application
+     *  @returns a promise the text of the svg
+     *
+     *  @remarks
+     *  https://console.configuration.product.dev.alertlogic.com/assets/static/logos/application-registry/okta.svg
+     */
+    async getLogo(imageName: string = 'default'): Promise<any> {
+
+        const applicationList = await this.client.get({
```
Comment (mcnielsen) [inline src/al-applications-client.ts]: Check your variable names here please

File: src/al-applications-client.ts [hunk 2]
```diff
@@ -37,6 +37,28 @@ export class AlApplicationsClientInstance {
...
+     * /assets/static/logos/application-registry/{imageName}.svg
+     * "https://console.configuration.product.dev.alertlogic.com/assets/static/logos/application-registry/okta.svg"
+     *
+     *  @param imageName this is the Id of the application
+     *  @returns a promise the text of the svg
+     *
+     *  @remarks
+     *  https://console.configuration.product.dev.alertlogic.com/assets/static/logos/application-registry/okta.svg
+     */
+    async getLogo(imageName: string = 'default'): Promise<any> {
```
Comment (scottarver) [inline src/al-applications-client.ts]: -1 for any

File: src/al-applications-client.ts (hunk 1 above)
Comment (scottarver) [inline src/al-applications-client.ts]: you can do `this.client.get({`
no variable names needed

File: src/al-applications-client.ts (hunk 2 above)
Comment (mcnielsen) [reply src/al-applications-client.ts]: Should be `string` since you're expecting a textual response.

**Comment by scottarver** [comment]
is this the best way to return and load an image?

whats wrong with using the `img` tag and `src` attribute?

**Comment by mcnielsen** [comment]
> is this the best way to return and load an image?
> 
> whats wrong with using the `img` tag and `src` attribute?

The images in this case will be of type `image/svg+xml`, and we need to use a fallback "default logo" image in cases where a specific application doesn't have an image associated with it.

File: src/al-applications-client.ts (hunk 1 above)
Comment (jleon-alertlogic) [reply src/al-applications-client.ts]: oops, applicationList was an error due copy/paste, good catch, thx

File: src/al-applications-client.ts (hunk 2 above)
Comment (jleon-alertlogic) [reply src/al-applications-client.ts]: you are right, thx

**Comment by scottarver** [comment]
> > is this the best way to return and load an image?
> > whats wrong with using the `img` tag and `src` attribute?
> 
> The images in this case will be of type `image/svg+xml`, and we need to use a fallback "default logo" image in cases where a specific application doesn't have an image associated with it.

loading via text feels wrong, but i can see why.

`img` tag can take an `svg`. 

and there is this
```
<img [src]="pic" (error)="setDefaultPic()">

setDefaultPic() {
  this.pic = "assets/images/my-image.png";
}
```

**Comment by mcnielsen** [comment]
@jleon-alertlogic  You might want to check out Jason's suggestion for dealing with images -- it might be cleaner all around than loading/defaulting manually.

**Comment by jleon-alertlogic** [comment]
I removed the getLogo() method

**Pull request merged**